mlir/test/fusion/e2e/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Dependencies for the E2E test only
+set(FUSION_E2E_DEPENDS
+  FileCheck count not
+  rocmlir-opt
+  rocmlir-gen
+  rocmlir-driver
+  mlir-cpu-runner
+  mlir_rocm_runtime
+  conv-validation-wrappers
+  e2e_tests
+  fusion-e2e-tests
+)
+
+set(E2E_DIR ${CMAKE_CURRENT_BINARY_DIR})
+set(E2E_GEN_SCRIPT "${CMAKE_CURRENT_SOURCE_DIR}/generate-fusion-tests.py")
+set(CONFIG_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(LOCAL_CONFIG_FILE_SRC ${CMAKE_CURRENT_SOURCE_DIR}/lit.local.cfg)
+set(LOCAL_CONFIG_FILE_BIN ${CMAKE_CURRENT_BINARY_DIR}/lit.local.cfg)
+
+# Add a command to copy lit.local.cfg
+add_custom_command(OUTPUT ${LOCAL_CONFIG_FILE_BIN}
```

Comment:
It seems that the fusion e2e tests are not guarded by any macros. So you do not need a `lit.local.cfg` to control when to run the tests. 
And this `add_custon_command()` does not have any effect because `${LOCAL_CONFIG_FILE_BIN}` is not a target.

Reply:
I copied a lot of the cmake files from other test sets.  I think I need to reexamine them.

Reply:
Ultimately I did need a lit.local.cfg, and this custom-command still didn't do anything, and now I copy lit.local.cfg in generate-fusion-tests.py.

Reply:
That shouldn't be it.

You might need to depend on the copy, or on the output, or otherwise get the command into CMake's dependency chain.